components/StoreLocatorModal.vue

```diff
@@ -147,7 +127,7 @@ export default {
     })
 
     const searchByZipCode = async () => {
-      await searchStoreLocations(zipCodeInput.value)
+      await searchStoreLocations({ filter: `geo near(${zipCodeInput.value},160934)` })
```

Comment:
Instead of hard coding variables use constants

Reply:
No constants file present